src/main/java/uk/gov/hmcts/reform/wataskmonitor/schedulers/TaskConfiguratorScheduler.java

```diff
@@ -32,10 +32,12 @@ public TaskConfiguratorScheduler(CamundaService camundaService,
 
     @Scheduled(fixedRateString = "${task.configurator.scheduling.fixedRate}")
     public void runTaskConfigurator() {
-        log.info("Task configurator starts...");
+        log.info("Starting task configurator.");
         String serviceToken = authTokenGenerator.generate();
         List<CamundaTask> camundaTasks = camundaService.getUnConfiguredTasks(serviceToken);
-        taskConfigurationService.configureTasks(camundaTasks, serviceToken);
-        log.info("Task configurator ends...");
+        if (!camundaTasks.isEmpty()) {
```

Comment:
don't think we need that if statement, since there is a forEach loop so if there are no tasks, is not doing anything.

Reply:
There's a log inside of there which was confusing me when there are 0 tasks as it said         "Configuring tasks..." but there was no tasks

Reply:
in that case, wouldn't be better to update the log?

Reply:
The log is helpful it doesn't hurt having a safe check before